Source/options.cpp

```diff
@@ -229,8 +229,10 @@ void LoadOptions()
 #else
 	sgOptions.Graphics.bFullscreen = true;
 #endif
-#if !defined(USE_SDL1)
+#if !defined(VIRTUAL_GAMEPAD) && !defined(USE_SDL1)
 	sgOptions.Graphics.bUpscale = GetIniBool("Graphics", "Upscale", true);
+#elif defined(VIRTUAL_GAMEPAD) && !defined(USE_SDL1)
+	sgOptions.Graphics.bUpscale = true;

```

Comment:
I'm a bit unsure if we should force this on just because it might have a negative effect on a partially related but optional feature :thinking:

Reply:
I'm forcing it on because the render API isn't used unless the upscale flag is set.
https://github.com/diasurgical/devilutionX/blob/6dbe4340458bfea947b7b52772252ec715dd7052/Source/utils/display.cpp#L220-L231

If I don't force it on, then I have to fall back on `SDL_BlitScaled()`, which is tricky because of how I have to shim the rendering call into `RenderPresent()`. Or, I suppose, we could simply not render the virtual gamepad.

Reply:
After thinking about it for a bit, I decided it wasn't all that tricky after all.

Reply:
Yeah I was thinking to opt for not rendering it. Some one with a crappy Android gamepad (emulating touch events) might even see it as a feature :joy: :joystick: